napari/_tests/test_viewer.py

```diff
@@ -12,8 +12,16 @@
 )
 from napari.utils._tests.test_naming import eval_with_filename
 
+viewer_methods = set(super(Viewer, Viewer).class_keymap.values())
+viewer_methods.update(Viewer.class_keymap.values())
 
-def test_viewer(make_napari_viewer):
+
+def test_len_methods_viewer():
```

Comment:
Can you add a comment doc string about what the `18` is? (I guess number of keybindings on the viewer).